bugtool/dump/file.go

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package dump
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// File implements a Task that will copy a file into a dump.
+type File struct {
+	base               `mapstructure:",squash"`
+	Src                string
+	ExcludeObjectFiles string `mapstructure:"exclude_object_files"`
```

Comment:
Likewise, rather than a specific feature for ExcludeObjectFiles, what about a negative-match regex or glob?